### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by presbrey**
## We want to ensure high quality of the packages. Make sure that you've checked the boxes below before sending a pull request.

- [x] I have read the [Contribution Guidelines](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#contribution-guidelines)
- [x] I have read the [Maintainers Note](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#maintainers)
- [x] I have read the [Quality Standards](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#quality-standards)

_Not every repository (project) will require every option, but most projects should. Check the Contribution Guidelines for details._

- [x] The repo documentation has a pkg.go.dev link.
- [x] The repo documentation has a coverage service link.
- [x] The repo documentation has a goreportcard link.
- [x] The repo has a version-numbered release and a go.mod file.
- [x] The repo has a continuous integration process that automatically runs tests that must pass before new pull requests are merged.
- [x] Continuous integration is used to attempt to catch issues prior to releasing this package to end-users.

## Please provide some links to your package to ease the review

- [x] forge link (github.com, gitlab.com, etc): https://github.com/presbrey/go-multiproxy
- [x] pkg.go.dev: https://pkg.go.dev/github.com/presbrey/go-multiproxy
- [x] goreportcard.com: https://goreportcard.com/report/github.com/presbrey/go-multiproxy
- [x] coverage service link ([codecov](https://codecov.io/), [coveralls](https://coveralls.io/), etc.): https://app.codecov.io/gh/presbrey/go-multiproxy/

## Pull Request content

- [x] The package has been added to the list in alphabetical order.
- [x] The package has an appropriate description with correct grammar.
- [x] As far as I know, the package has not been listed here before.

## Category quality

_Note that new categories can be added only when there are 3 packages or more._

Packages added a long time ago might not meet the current guidelines anymore. It would be very helpful if you could check 3-5 packages above and below your submission to ensure that they also still meet the Quality Standards.

Please delete one of the following lines:

- [ ] The packages around my addition still meet the Quality Standards.
- [x] I removed the following packages around my addition: (please give a short reason for each removal)

#### golibwireshark: tests don't work, no CI, no coverage, no go.mod, no report card, no vX.Y.Z tag
```
golibwireshark$ go test -v
# github.com/sunwxg/golibwireshark.test
/usr/local/src/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: /tmp/go-link-2602567090/000003.o:/golibwireshark/include/lib.h:23: multiple definition of `cfile'; /tmp/go-link-2602567090/000001.o:/golibwireshark/./include/lib.h:23: first defined here
/usr/bin/ld: /tmp/go-link-2602567090/000003.o:/golibwireshark/include/lib.h:24: multiple definition of `pdh'; /tmp/go-link-2602567090/000001.o:/golibwireshark/./include/lib.h:24: first defined here
collect2: error: ld returned 1 exit status
```

Thanks for your PR, you're awesome! :sunglasses:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1951,11 +1951,11 @@ _Libraries for working with various layers of the network._
 - [gnet](https://github.com/fish-tennis/gnet) - `gnet` is a high-performance networking framework,especially for game servers.
 - [gNxI](https://github.com/google/gnxi) - A collection of tools for Network Management that use the gNMI and gNOI protocols.
 - [go-getter](https://github.com/hashicorp/go-getter) - Go library for downloading files or directories from various sources using a URL.
+- [go-multiproxy](https://github.com/presbrey/go-multiproxy) - Library for making HTTP requests through a pool of proxies offering fault tolerance, load balancing, automatic retries, cookie management, and more, via http.Get/Post replacement or http.Client RoundTripper drop-in
```
Comment (yassinebenaid) [inline README.md:1954]: The exported types/functions are missing doc comments.

_CONTRIBUTING.md_
>be thoroughly documented (README, pkg.go.dev doc comments, etc.) in the English language, so everyone is able to understand the project's intention and how it works. All public functions and types should have a Go-style documentation header;

**Review by yassinebenaid — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (presbrey) [reply README.md:1954]: Thank you for the quick review. Sorry I missed that, the exported functions work like http.{Get,Post} but its best to have the doc comments specified. Please see commit at: https://github.com/presbrey/go-multiproxy/commit/d8ce296a6a60e1bc6e7dd56d854d680ff5a2e7d0

Comment (yassinebenaid) [reply README.md:1954]: LGTM 🔥 .

I'll merge as soon as the changes are released.

Comment (presbrey) [reply README.md:1954]: Great! Thank you, here is the release:
> https://github.com/presbrey/go-multiproxy/releases/tag/v1.0.1

**Pull request merged**